samples/cpp/videocapture_realsense.cpp

```diff
@@ -0,0 +1,37 @@
+#include "opencv2/videoio.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+
+using namespace cv;
+using namespace std;
+
+int main()
+{
+   VideoCapture capture(CAP_INTELPERC);
+   for(;;)
+   {
+      double min;
+      double max;
+
+      Mat depthMap;
+      Mat image;
+      Mat irImage;
+      Mat adjMap;
+
+      capture.grab();
+      capture.retrieve(depthMap,CAP_INTELPERC_DEPTH_MAP);
+      capture.retrieve(image,CAP_INTELPERC_IMAGE);
+      capture.retrieve(irImage,CAP_INTELPERC_IR_MAP);
+
+      cv::minMaxIdx(depthMap, &min, &max);
+      depthMap.convertTo(adjMap,CV_8UC1, 255 / (max-min), -min);
```

Comment:
It is time to learn about `cv::normalize()`.

Reply:
> 
Hello dear @alalek 

Intel perceptual camera is very incompatible with current realsense. For various reasons and parameters of the videocapture_intelperc example realsense and realsenseid do not work.

Reply:
> It is time to learn about `cv::normalize()`.

doing...

Reply:
> > It is time to learn about `cv::normalize()`.
> 
> doing...

Thank you @alalek , even if the PR is not approved... I always learn from you!

Hugs.

Reply:
> For various reasons and parameters of the videocapture_intelperc example realsense and realsenseid do not work.

It makes sense to fix the sample and make it compatible with Realsense cameras.

Reply:
done 
![opencv](https://user-images.githubusercontent.com/675645/146277206-b59ef5c2-b85f-4689-93ab-2919b63b935b.jpg)

Reply:
Super @alalek  

Not even librealsense is fully compatible with Intel Perceptual (hardware is very obsolete)

I have the Intel family, see below...

![realsense](https://user-images.githubusercontent.com/675645/146277596-3931ca65-8ecd-4b3b-baa1-c0abe5b31d51.jpg)

Reply:
If old sample is obsolete, then it makes sense to remove that sample completely and keep the new one.

Reply:
ok

Reply:
done

Reply:
There is reference on removed sample from obsolete tutorial which is patched in previous PR.
So it makes sense to update this too.

Reply:
Very well remembered. doing...